### DISCUSSION TITLE
Please help with sending POST Webhook

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#2172] [Help]
Hey all,

I am new to it and never used webhooks before. I would like to convert a document (web URL) to PDF.
The API I am trying to use for it: https://pdflayer.com/documentation#quickstart

As I understand, the flow to work with it in BB is:

- Create new Automation with trigger App Action
- Add Send Webhook and setup it
- Call the Automation from the app 
_- Get PDF file_

The screens:

<img width="1792" alt="Screenshot 2021-07-29 at 13 47 15" src="https://user-images.githubusercontent.com/53024832/127480267-5734aabc-c7dd-4037-909a-b1c73ee0fd6a.png">

<img width="1792" alt="Screenshot 2021-07-29 at 13 46 20" src="https://user-images.githubusercontent.com/53024832/127480289-abbdae10-a464-45a7-b540-51c3bfea1026.png">

Something is wrong here, it does not work.

I successfully tested this service sending GET request with this test URL and I would use the GET method in my app just as a LInk, BUT when GET is using this API additionally requires secret_key in MD5 (url and secret word), which I unable to encode in BB, did not find such Handlebar for it. So, the only way to use this API in bb is sending POST Webhook..  Can anyone help with this please?

**Comment by shogunpurple** [comment]
Hi @sergey-dme - a few questions on this one:

- Does BB actually hit the API at all? An easy way to check this is to use a "Save Row" automation to save the response into a text field, just to see what happens. The alternative is to use the server log action, which will log the value to the backend, but will require you to look at the docker logs after.

<img width="543" alt="Screenshot 2021-07-29 at 14 46 22" src="https://user-images.githubusercontent.com/11256663/127503326-60585664-365c-488c-88c5-8fdd1e4f187c.png">

Once we know whether the API request is actually being sent, we can debug further.
